src/core/modules/StateEvaluator.ts

```diff
@@ -132,6 +132,8 @@ export class DefaultEvaluationOptions implements EvaluationOptions {
   walletBalanceUrl = 'http://nyc-1.dev.arweave.net:1984/';
 
   mineArLocalBlocks = true;
+
+  throwOnInternalWriteError = true;
```

Comment:
@jakub-wojciechowski the default should be set to `true`, right?

Reply:
Yes